code/game/mecha/combat/combat.dm

```diff
@@ -2,9 +2,6 @@
 	force = 30
 	internal_damage_threshold = 50
 	damage_absorption = list("brute"=0.7,"fire"=1,"bullet"=0.7,"laser"=0.85,"energy"=1,"bomb"=0.8)
-	var/am = "d3c2fbcadca903a41161ccc9df9cf948"

```

Comment:
Not used, unless you want to dig the history of the Mecha folder, we'll never know.